modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java

```diff
@@ -68,16 +67,17 @@
         String CACHES = "cacheName1,...,cacheNameN";
 
         String desc = "Verify whether there are inconsistent entries for the specified caches " +
-            "and print out the differences if any. Fix inconsistency if " + FIX_MODE + "argument is presented. " +
```

Comment:
Don't forget to update control.sh_cache_help.output

Reply:
fixed